lib/jazzy/sourcekitten.rb

```diff
@@ -52,8 +52,33 @@ def self.make_doc_urls(docs, parents)
       end
     end
 
+    def self.assert_xcode_location
+      expected_xcode_select_path =
+        Pathname('/Applications/Xcode.app/Contents/Developer')
+      return if xcode_developer_directory == expected_xcode_select_path
+      raise 'Please install Xcode 6.1 or 6.2 in ' \
```

Comment:
Especially as it is common to have multiple Xcode versions nowadays (with betas and all), and people don't always want to be forced to use a given strict name for their Xcode apps, I feel like creating a symlink should be also be an alternative listed in the exception message (if using a symlink still actually works obviously)?